CENNZnet-API/JSON-RPC-API.md

```diff
@@ -7,11 +7,22 @@ Use a cURL like the one below to call any of these JSON-RPC requests from CENNZn
 Just change the method and params to what you're looking for, and it should return a JSON response.
 
 #### Example request
-`
+```
 curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "nft_getTokenInfo", "params": [0,0,0]}' https://cennznet.unfrastructure.io/public
```

Comment:
tip: can add some `\` to break up the lines
```
curl -H "Content-Type: application/json" \
  -d '{"id":1, "jsonrpc":"2.0", "method": "nft_getTokenInfo", "params": [0,0,0]}' \
  https://cennznet.unfrastructure.io/public
```

Reply:
oh well looks like our service doesn't like new lines :P happy to leave the example code as it is :)